research/RSOC.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<p>OK, jak zmrużysz oczy to wygląda w miarę jak mrówki - SOCa tam nie widzę na pierwszy rzut oka, ale spoczko ;) I tak bym nie wiedział po czym go poznać.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SocSIM/SocSIM/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>